src/main/java/com/zendesk/maxwell/replication/MaxwellReplicator.java

```diff
@@ -214,7 +238,6 @@ private RowMapBuffer getTransactionRows() throws Exception {
 				continue;
 			}
 
-			setReplicatorPosition((AbstractBinlogEventV4) v4Event);
```

Comment:
Remind me why this went away - doesn't it mean that on restart, we'll jump back to the beginning of the most recent transaction? If we're in the middle of `getTransactionRows()` when that happens, we'll get very confused.